### PR TITLE
[R4R] Update release link

### PR DESCRIPTION
**Pull request by qinglin89**
### Description

The old `release` link is Invalid.
Update the new one https://github.com/bnb-chain/bsc/releases/tag/v1.1.9

### Rationale

N/A

### Example

N/A

### Changes

Notable changes:
- README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -124,7 +124,7 @@ This command will:
 
 Steps:
 
-1. Download the binary, config and genesis files from [release](https://github.com/binance-chain/bsc/releases/download/v1.0.0-alpha.0/binary.zip), or compile the binary by `make geth`. 
+1. Download the binary, config and genesis files from [release](https://github.com/bnb-chain/bsc/releases/download/v1.1.8/mainnet.zip), or compile the binary by `make geth`.
```
Comment (guagualvcha) [inline README.md:127]: Here is to download the binary, while mainnet.zip is just config.

Comment (qinglin89) [reply README.md]: updated

**Review by brilliant-lx — APPROVED** [review]
LGTM

**Review by brilliant-lx — APPROVED** [review]
LGTM

**Review by forcodedancing — APPROVED** [review]

**Pull request closed without merging**